src/components/Initiatives/InitiativeDetailsPage/InitativeFilterDialog/InitiativeFilterDialog.tsx

```diff
@@ -27,7 +27,7 @@ import {
   ruleName,
 } from '../../../../api/types';
 import { FilterProvider, useFilter } from '../../../FilterCard/useFilter';
-import { mapValues } from 'lodash';
+import { capitalize, mapValues } from 'lodash';
```

Comment:
Need to remove the unused import now
```suggestion
import { mapValues } from 'lodash';
```